content/legal/release-policy.md

```diff
@@ -44,36 +44,26 @@ requirements of ASF policy on releases as described below, validate all
 cryptographic signatures, compile as provided, and test the result on their
 own platform.
 
-Release votes SHOULD remain open for at least 72 hours. See the next
-[Expedited Releases](#expedited-releases) section when considering a reduced
-voting period.
+Unless there are strong reasons to expedite a release, as described below,
+release votes SHOULD remain open for at least 72 hours, to give PMC members
+enough time to participate.
 
 #### Expedited Releases {#expedited-releases}
-As stated above, the normal policy for releases is to allow 72 hours for
-release reviews and votes, however the review/voting period for a release
-can be reduced in exceptional circumstances.
+The review and voting period for a release can be reduced in
+exceptional circumstances.
 
-ASF projects are made up of distributed teams, in multiple time zones and volunteers
-with lives and jobs and the rationale behind 72 hours is to try and give all
-members of a project the opportunity to take part in the decision to release.
+A typical example is a release that fixes publicly known or
+critical exploitable security issues.
 
-The most obvious example of an exceptional circumstance would be for a fix for a
-publicly known or critical, easily exploitable security issue. Everyone will probably have a different definition
-of what an exceptional circumstance is, but ultimately it is down to individual
-PMCs to decide for their project. 
```

Comment:
A key motivator for the previous change was to provide an explicit example.

Reply:
Changed this to clarify that the PMC can decide, which says the same as previously but in a more precise & concise way.